reloadRecipe.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+echo "Run this script from the same directory as your recipe metadata.json"
+echo "Provide an environment shortname which is at least Flow V5.11."
```

Comment:
If this message described how to provide the shortname, that would eliminate the need to read the script.

Reply:
Done, thanks